storybook/elm-webpack-loader-fix.js

```diff
@@ -0,0 +1,219 @@
+"use strict"
+
+var fs = require("fs")
+var path = require("path")
+var loaderUtils = require("loader-utils")
+var elmCompiler = require("node-elm-compiler")
+var yargs = require("yargs")
+
+var runningInstances = 0
+var alreadyCompiledFiles = []
+
+var defaultOptions = {
+  cache: false,
+  forceWatch: false,
```

Comment:
This was the first bug I found. Normally you don't need to include this, because `node-elm-compiler` infers that webpack is in watch mode by parsing the command line arguments (https://github.com/elm-community/elm-webpack-loader/blob/master/index.js#L79), but Storybook does not use the dev server, it has it's own custom server, presumably with watch mode configured somewhere. Fixing this caused the problem where too many files were being watched.